lib/async-await.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+/**
+ * @param  {} file
+ * @return stringData 
+ */
+async function readFile1(file) {
```

Comment:
I would not have a number in the function name unless it relates to what the function does